reactive/kubernetes_master.py

```diff
@@ -1374,9 +1374,26 @@ def send_data():
                                    key_path=client_key_path)
 
 
+def check_flags(*flags):
+    """Check if any of the provided flags missing and return them if so."""
```

Comment:
Suggestions:
* change the func name to get_unset_flags(*flags), which sounds more accurate?
* """Return any non-active flag shared as an argument""".
* add something like:
```
:param flags: list of reactive flags
:type flags: non-keyword args, str
:returns: list of unset flags filtered from the parameters shared
:rtype: List[str]
```